app/views/create-tag.html

```diff
@@ -5,7 +5,7 @@
 
 <h1>Create a tag for image ID <a href="#/image/{{imageId}}">{{imageId | limitTo: 12}}</a></h1>
 
-<form name="createTagForm" novalidate class="form-horizontal" role="form" class="css-form">
+<form ng-hide="appMode.browseOnly" name="createTagForm" novalidate class="form-horizontal" role="form" class="css-form">
```

Comment:
Happy New Year! I'm sorry but I cannot merge this when the buttons flicker and then disappear. I will try to look into this when there's time. Please leave this PR open and thank you for your work so far. I know it is frustrating to have one's PRs rejected but please bear with me.